pkg/api/delete.go

```diff
@@ -2,13 +2,14 @@ package api
 
 import (
 	"fmt"
+	delete2 "github.com/timescale/promscale/pkg/pgmodel/delete"
```

Comment:
This name `delete2` sticks out like a sore thumb. 
Makes the code confusing, personally I would need to try to find out what the `2` stands for.

```suggestion
	deletePkg "github.com/timescale/promscale/pkg/pgmodel/delete"
```
Also, imports need to be properly grouped and sorted.

Reply:
Sorry, that was by the code editor. I will fix all these.